### ISSUE TITLE
[Screen Reader – BotKit – Docs]: In Scan Mode, Unable to navigate the headings in the pages which are present under Docs.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#6]
**User Experience:**
Users who rely on screen readers will affect to navigate, if user is unable to navigate the headings while navigating using scan mode.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to “Docs” link and select it.
Step 3: Navigate to “Introduction” link and select it.
Step 4: Enable Narrator from the desktop.
Step 5: Enable Scan Mode (Caps Lock + Spacebar)
Step 6: Navigate to the Introduction page using Scan mode up/down arrow keys.
Step 7: Verify whether user is able to navigate the headings in Scan mode.

**Actual Result:**
Unable to navigate the headings in the page while navigating Scan mode using Up/down arrow keys.

**Expected Result:**
User should be able to navigate the headings in the page while navigating Scan mode using Up/down arrow keys.

**Suggestion**: Need to remove "aria-hidden="true" either aria-hidden="false" should be defined.

**Note**: Same issue repros in all the pages which are present under “Docs” link.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Ba12f3779-bdf1-40c2-bbcc-0e314ea94858%7D]()
![MAS4 2 1_BotKit_scan mode issue](https://user-images.githubusercontent.com/48272005/54422002-8e6e2c80-4733-11e9-8b28-c3ff6e9420ba.png)
[MAS4.2.1_BotKit_Scan Mode issue.zip](https://github.com/howdyai/botkit-website/files/2970332/MAS4.2.1_BotKit_Scan.Mode.issue.zip)

**Comment by benbrown** [comment]
aria-hidden attribute has been removed from all of these links.

**Comment by v-gasikk** [comment]
Issue is Not Fixed.
aria-hidden attribute is still visible in the code, because In Scan Mode, Unable to navigate the headings in the pages which are present under Docs.

**Comment by benbrown** [comment]
@v-gasikk  This was due to a bug in our page regeneration tool.

The aria-hidden attribute has been removed again and should stay gone this time!

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
Now aria-hidden attribute has removed.
